[2023-08-14T06:09:31.830+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_dag.fail scheduled__2023-08-06T00:00:00+00:00 [queued]>
[2023-08-14T06:09:31.846+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_dag.fail scheduled__2023-08-06T00:00:00+00:00 [queued]>
[2023-08-14T06:09:31.846+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-08-14T06:09:31.847+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-08-14T06:09:31.848+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-08-14T06:09:31.867+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): fail> on 2023-08-06 00:00:00+00:00
[2023-08-14T06:09:31.872+0000] {standard_task_runner.py:55} INFO - Started process 2221 to run task
[2023-08-14T06:09:31.875+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_dag', 'fail', 'scheduled__2023-08-06T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/data_dag.py', '--cfg-path', '/tmp/tmpns8uygm1']
[2023-08-14T06:09:31.876+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask fail
[2023-08-14T06:09:31.892+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:400: DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
  engine_args["encoding"] = conf.get("database", "SQL_ENGINE_ENCODING", fallback="utf-8")

[2023-08-14T06:09:31.951+0000] {task_command.py:388} INFO - Running <TaskInstance: data_dag.fail scheduled__2023-08-06T00:00:00+00:00 [running]> on host 7dfb041806a2
[2023-08-14T06:09:32.036+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_dag
AIRFLOW_CTX_TASK_ID=fail
AIRFLOW_CTX_EXECUTION_DATE=2023-08-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-06T00:00:00+00:00
[2023-08-14T06:09:32.037+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-08-14T06:09:32.038+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n                valid=06\n                if [ $(($valid % 2)) == 1 ]; then\n                        exit 1\n                else\n                        exit 0\n                fi\n            ']
[2023-08-14T06:09:32.048+0000] {subprocess.py:86} INFO - Output:
[2023-08-14T06:09:32.050+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-14T06:09:32.081+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=data_dag, task_id=fail, execution_date=20230806T000000, start_date=20230814T060931, end_date=20230814T060932
[2023-08-14T06:09:32.128+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-08-14T06:09:32.146+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
